Jordan/rename ci jobs

jbibla opened the pull request:
i have no idea how any of this stuff actually works so this is just an experiment 🙃

faboweb commented:
<!-- NETLIFY DEPLOY COMMENT GENERATED BY ACTIONS_NETLIFY -->
🚀 Deployed on https://5f368c9fc614e9cd1b8d9790--lunieio.netlify.app

Bitcoinera commented on .github/workflows/publish.yml:
```diff
@@ -1,4 +1,4 @@
-name: Publish to master
+name: Publish Jobs
```
I actually find "Publish to master" more eloquent / clear

jbibla replied:
it's the "Publishing Workflow" - not a specific job

Bitcoinera commented on app/.codecov.yml:
```diff
@@ -1,14 +1,14 @@
+#   status:
+#     project: no
+#     patch: yes
+#     changes: no
```
Good bye codecov :wave:

jbibla commented on .github/workflows/publish.yml:
```diff
@@ -1,4 +1,4 @@
-name: Publish to master
+name: Publish Jobs
```
```suggestion
name: Publishing Workflow
```

jbibla commented on .github/workflows/app_test.yml:
```diff
@@ -1,4 +1,4 @@
-name: App testing
+name: App Jobs
```
```suggestion
name: App Testing Workflow
```

jbibla commented on .github/workflows/app_deploy.yml:
```diff
@@ -1,4 +1,4 @@
-name: Publish on Netlify
+name: Deploy on Netlify
```
```suggestion
name: App Deployment Workflow
```

jbibla commented on .github/workflows/api_test.yml:
```diff
@@ -1,4 +1,4 @@
-name: API testing
+name: API Jobs
```
```suggestion
name: API Testing Workflow
```

faboweb commented:
should we really ignore testing?

jbibla commented:
> should we really ignore testing?

what do you mean?

Bitcoinera commented on .github/workflows/app_deploy_production.yml:
```diff
@@ -1,4 +1,4 @@
-name: Publish on Netlify
+name: App Production Deployment Workflow
```
Nice!

Bitcoinera commented on app/package.json:
```diff
@@ -6,7 +6,7 @@
     "build": "vue-cli-service build --no-unsafe-inline --skip-plugins pwa --mode production",
     "build:dev": "vue-cli-service build --no-unsafe-inline --skip-plugins pwa --mode development",
-    "buildsentry": "vue-cli-service build --no-unsafe-inline --skip-plugins pwa --modern --mode production && chmod 755 tasks/sentry.sh && ./tasks/sentry.sh",
+    "build:sentry": "vue-cli-service build --no-unsafe-inline --skip-plugins pwa --modern --mode production && chmod 755 tasks/sentry.sh && ./tasks/sentry.sh",
```
Nice

Bitcoinera commented:
Please, resolve the conflicts, but must say I really like the renaming. Adds much more clarity to the whole

faboweb commented:
needs a merge. please remove any change that is not renaming relevant:
- readd codecov
- readd polkadot e2e

jbibla commented:
want to confirm 

- i added yarn --fix in a couple places - will this work as expected?
- i deleted some workflow folders from within the specific projects - they were unused, right?

Bitcoinera commented:
> * i added yarn --fix in a couple places - will this work as expected?

You mean `yarn lint --fix`? Yes, it should work

> * i deleted some workflow folders from within the specific projects - they were unused, right?

Yeah, they weren't used anymore... I believe, actually. @faboweb ?
